File: examples/js/youtube/search/searchForVideos.js
const { YoutubeService } = require("m3o/youtube");

// Search for videos on YouTube
async function searchForVideos() {
  let youtubeService = new YoutubeService(process.env.MICRO_API_TOKEN);
  let rsp = await youtubeService.search({
    query: "donuts",
  });
  console.log(rsp);
}

searchForVideos();

File: examples/go/youtube/search/searchForVideos.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/youtube"
)

// Search for videos on YouTube
func SearchForVideos() {
	youtubeService := youtube.NewYoutubeService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := youtubeService.Search(&youtube.SearchRequest{
		Query: "donuts",
	})
	fmt.Println(rsp, err)
}
